### ISSUE TITLE
*.h files from contrib/ directory shouldn't be copied into package

### ISSUE THREAD

**Issue by arsonaut** [conan-community/conan-zlib#13]
In `package()` method all header files are copied from source folder recursively, including that:

    ./zlib-1.2.8/contrib/blast/blast.h
    ./zlib-1.2.8/contrib/infback9/infback9.h
    ./zlib-1.2.8/contrib/infback9/inffix9.h
    ./zlib-1.2.8/contrib/infback9/inflate9.h
    ./zlib-1.2.8/contrib/infback9/inftree9.h
    ./zlib-1.2.8/contrib/iostream/zfstream.h
    ./zlib-1.2.8/contrib/iostream2/zstream.h
    ./zlib-1.2.8/contrib/iostream3/zfstream.h
    ./zlib-1.2.8/contrib/minizip/crypt.h
    ./zlib-1.2.8/contrib/minizip/ioapi.h
    ./zlib-1.2.8/contrib/minizip/iowin32.h
    ./zlib-1.2.8/contrib/minizip/mztools.h
    ./zlib-1.2.8/contrib/minizip/unzip.h
    ./zlib-1.2.8/contrib/minizip/zip.h
    ./zlib-1.2.8/contrib/puff/puff.h

But they don't needed, and most important, their presence will hide lack of (for example) minizip library until linking.

As `conans.ConanFile.copy()` method doesn't have option for non-recursive copy, maybe best solution will be to delete `contrib/` folder in `source()` method.

**Comment by liberforce** [comment]
Seem this is fixed in 1.2.11.
The only headers I can find are:
```
crc32.h
inffixed.h
trees.h
inftrees.h
zutil.h
deflate.h
inffast.h
gzguts.h
zlib.h
gzlog.h
inflate.h
zconf.h
```

**Comment by SSE4** [comment]
This issue was moved to conan-community/community#61
